### ISSUE TITLE
Rewind Time

### ISSUE THREAD

**Issue by Wukarditd** [absoIute/Mega-Hack-Pro-Future#77]
I have no idea if this is even possible but if it is, make it possible to rewind the level and the player position.

**Comment by Brittank88** [comment]
What would be the purpose of this?

Technically you can run the level in reverse but if you mean running it in reverse and remembering exactly where the player came from that's a whole different story.

**Comment by Wukarditd** [comment]
Yeah, like remembering the player's positions and rewinding everything that happened. I have seen it in emulators and i was wondering if you could do the same in GD.

**Comment by Brittank88** [comment]
This can probably be accomplished but I can't imagine it would be easy at all. What use would this even have?

**Comment by absoIute** [comment]
sounds like a cool feature for botting/macros, so may add something like this (or a reverse Frame-Advance-like thing) if I end up adding native botting

**Comment by defaultacc30** [comment]
> I have no idea if this is even possible but if it is, make it possible to rewind the level and the player position.

this can be done by the variables tab already so it is easy

**Comment by absoIute** [comment]
Implemented in v7.1
